docs/source/tutorial/interfaces.rst

```diff
@@ -360,7 +358,7 @@ which reads a number from the console, returning a value of the form
          else pure Nothing
 
 If we then use it to write a function to read two numbers, returning
-``Nothing`` if neither are valid, then we would like to pattern match
+``Nothing`` if either is invalid, we would like to pattern match
```

Comment:
Should potentially be "neither is valid", but it is clearer as "neither _ valid" rather than "either _ invalid".

Reply:
Done.